app/uk/gov/hmrc/fhregistrationfrontend/models/ModelEncryption.scala

```diff
@@ -48,4 +50,26 @@ object ModelEncryption {
     )
   }
 
+  def encryptSessionCache(
+    sessionCache: SummaryConfirmation
+  ): (String, Option[String], Option[WithdrawalReason], Option[DeregistrationReason]) =
+    (
+      sessionCache.id,
+      sessionCache.summaryForPrintKey,
+      sessionCache.withdrawalReason,
+      sessionCache.deregistrationReason
+    )
+
+  def decryptSessionCache(
```

Comment:
See https://github.com/hmrc/ei-frontend/pull/220/files#diff-db93cae2aeca63146de522b94b717eaa25ceb76d32be1e9ea266d9502f716380R31

Needs to actually decrypt